status-im/status-react
# feature #3953 - enabled mainnet on release builds, with a warning

fixes #3953 

### Summary:

Enabled mainnet on release builds as a non-default option. When users tries to switch to mainnet, they are shown a warning stating the experimental state of this option.

### Testing notes (optional):
The behavior added in this PR depends on whether or not the app is built in release or dev mode. Both need to be tested. (Will run a release build in jenkins and add the links here)

### Steps to test:
- Start a release build
- Check that network is not already set to mainnet
- Check that switching to mainnet is possible and that the warning is there

- Start a nightly build 
- Check that network is mainnet by default
- Check that switching from and to mainnet has no warnings

status: ready
